crowbar_framework/app/models/service_object.rb

```diff
@@ -722,14 +725,15 @@ def proposal_create(params)
     # When we create a proposal, it might be "invalid", as some roles might be missing
     # This is OK, as the next step for the user is to add nodes to the roles
     # But we need to skip the after_save validations in the _proposal_update
-    _proposal_update(proposal, false)
+    _proposal_update(@bc_name, base_id, proposal, false)
   end
 
   def proposal_edit(params, validate_after_save = true)
-    params["id"] = "bc-#{@bc_name}-#{params["id"] || params[:name]}"
+    base_id = params["id"] || params[:name]
```

Comment:
So now I have to wonder: why do we not accept `params[:name]` in proposal_create and accept it here?

(this is not your change, but would be good to understand so we can fix it if needed)